src/config/default-app-config.ts

```diff
@@ -324,4 +325,8 @@ export class DefaultAppConfig implements AppConfig {
     image: false,
     video: false
   };
+  info: InfoConfig = {
```

Comment:
Please copy the comments to this file to describe these new settings (just like all the settings above have comments to describe them)